modules/datadog-agent/providers.tf

```diff
@@ -58,10 +58,4 @@ data "aws_eks_cluster_auth" "kubernetes" {
   name = module.eks.outputs.eks_cluster_id
 }
 
-provider "helm" {
-  kubernetes {
-    host                   = local.enabled ? data.aws_eks_cluster.kubernetes[0].endpoint : null
-    token                  = local.enabled ? data.aws_eks_cluster_auth.kubernetes[0].token : null
-    cluster_ca_certificate = local.enabled ? base64decode(data.aws_eks_cluster.kubernetes[0].certificate_authority[0].data) : null
-  }
-}
+provider "utils" {}
```

Comment:
Do we use utils provider?

Reply:
Yes https://github.com/cloudposse/terraform-aws-components/pull/409/files#diff-9d977686018b77e00be742c079a4f3c3259a1d9266f8dc0c8036837064b10e78R57 uses it as part of the module now